spec/controllers/api/api_users_controller_spec.rb

```diff
@@ -5,12 +5,14 @@
 describe Api::ApiUsersController do
   include ControllerHelpers
 
-  before(:each) do
-    login_as(:bob)
+  before(:each) do |test|
+    login_as(:bob) unless test.metadata[:admin_test]
```

Comment:
didn't know you could do this with rspec! nicely researched guys !
but ... I don't like it ... sry :joy: 
it makes the code not really readable ...
there's also only two roles around api_users .... 1. my api users, 2. not mine
so no need for testing this against the other roles user, admin, conf_admin